### DISCUSSION TITLE
raspberry_pi4-opensuse-samba/

### DISCUSSION THREAD

**Discussion by giscus** [guinuxbr/guinuxbr.com#2] [Announcements]
# raspberry_pi4-opensuse-samba/

IntroductionHello, in this article, I’ll explain how to install openSUSE Tumbleweed (however, these instructions would probably work in openSUSE Leap) in Raspberry Pi 4, plus install and configure a basic Samba server to share files from an external HDD.
The first thing to do is install the openSUSE Tumbleweed image on your Raspberry Pi 4. Go to the openSUSE Wiki page for Raspberry Pi 4 and download the image that you like to use.

https://guinuxbr.com/raspberry_pi4-opensuse-samba/

**Comment by guinuxbr** [comment]
It would be nice to have an article on running Samba and other services in containers on the Raspberry Pi.
